encoding/json/decode.go

```diff
@@ -691,6 +701,18 @@ func toObject(valueJSON interface{}) jsonObject {
 	return v
 }
 
+func compositeTypeIDFromObject(valueJSON interface{}) cadence.CompositeTypeID {
+	object := toObject(valueJSON)
+
+	id := compositeTypeIdentifierKey
+	object.GetString(id)
```

Comment:
Is this to force a panic?

Reply:
Not sure I get what you mean?

Reply:
The value returned from `object.GetString(id)` is never used, so at first I thought these two lines might be redundant, but then I thought you might be checking for the existence of `compositeTypeIdentifierKey`. `GetString` will panic if it's not found